cmd/ferretdb/ping.go

```diff
@@ -95,46 +95,43 @@ func ping() {
 			RawQuery: values.Encode(),
 		}
 
-		urls = append(urls, u.String())
+		urls = append(urls, u)
 	}
 
 	if cli.Listen.Unix != "" {
 		l.Debugf("--listen-unix flag is set. Ping to %s will be performed.", cli.Listen.Unix)
 
-		urls = append(urls, "mongodb://"+url.PathEscape(cli.Listen.Unix))
+		urls = append(urls, &url.URL{Scheme: "mongodb", Path: url.PathEscape(cli.Listen.Unix)})
```

Comment:
I've actually tried to implement it like that before, but unfortunately `url.URL` is not meant for unix socket as it escapes `%` from the `url.PathEscape`. It doesn't escape '/' though, so it's not possible to not use `PathEscape` function.